pkg/ipamd/datastore/data_store.go

```diff
@@ -327,13 +344,13 @@ type CheckpointData struct {
 // in checkpoints.
 type CheckpointEntry struct {
 	IPAMKey
-	IPv4 string `json:"ipv4"`
+	IP string `json:"ip"`
```

Comment:
Agreed. Since, we don't support v4 to v6 dynamic upgrade I thought this would be Ok but it does break v4->v4 upgrade with this change. Will address it.